### PR TITLE
Enabled notifications on boshrelease pipeline

### PR DESCRIPTION
**Pull request by geofffranks**

### CODE REVIEW

File: boshrelease/pipeline.yml [hunk 1]
```diff
@@ -131,6 +140,13 @@ jobs:
 
             AWS_ACCESS_KEY:       (( grab meta.aws.access_key ))
             AWS_SECRET_KEY:       (( grab meta.aws.secret_key ))
+        on_failure:
+          put: notify
+          params:
+            channel:  (( grab meta.slack.channel ))
+            username: (( grab meta.slack.username ))
+            icon_url: (( grab meta.slack.icon ))
+            text:     (( grab meta.slack.test_failure ))
```
Comment (jhunt) [inline boshrelease/pipeline.yml]: Should this be using `build_failure` instead of `test_failure`?

File: boshrelease/pipeline.yml [hunk 2]
```diff
@@ -38,7 +39,8 @@ meta:
 
   slack:
     webhook:       (( param "Please specify your Slack Incoming Webhook Integration URL" ))
-    notification: '(( concat ":sadpanda: " meta.name " build failed!<br>URL-GOES-HERE" ))'
+    build_failure: '(( concat "$BUILD_PIPELINE_NAME: :airplane_arriving: <" meta.url "/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME| Failed to ship a new bosh release. :sad-panda:>" ))'
+    test_failure:  '(( concat "$BUILD_PIPELINE_NAME: :airplane_arriving: <" meta.url "/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME| Tests failed!>" ))'
```
Comment (jhunt) [inline boshrelease/pipeline.yml]: Can we make the emoji configurable (i.e. `meta.slack.emoji`)?  Also, why no sadpanda? :cry:

File: boshrelease/pipeline.yml (hunk 1 above)
Comment (geofffranks) [inline boshrelease/pipeline.yml]: This one shouldn't, but the other one should.

**Comment by geofffranks** [comment]
Addressed feedback

**Pull request merged**